docs/en/stack/ml/nlp/ml-nlp-elser.asciidoc

```diff
@@ -385,6 +385,35 @@ The following sections provide information about how ELSER performs on different
 hardwares and compares the model performance to {es} BM25 and other strong 
 baselines such as Splade or OpenAI.
 
+[discrete]
+[[version-overview]]
+=== Version overview
+
+The ELSER model has two main versions: ELSER V2 and ELSER V1.
+ELSER V2 has a **platform specific** version that is designed to run on Linux 
+with an x86-64 CPU architecture and a **platform agnostic or portable** version 
+that can be run on any platform.
+
+[discrete]
+==== ELSER V2
+
+The biggest change in ELSER V2 is the introduction of the first platform 
+specific ELSER model - that is, a model that can only run on Linux with an 
+x86-64 CPU architecture. The platform specific model is designed to work best on 
+newer Intel CPUs, but it works on AMD CPUs as well. It is recommended to use the 
+new platform specific Linux-x86-64 model for all new users of ELSER as it is 
+significantly faster than the platform agnostic (portable) model which can be 
+run on any platform. ELSER V2 produces significantly higher quality embeddings 
+than ELSER V1. Regardless of which ELSER V2 model you use (platform specific or 
+platform agnostic (portable)), the particular embeddings produced are the same.
+
+[discrete]
+==== ELSER V1
```

Comment:
In this section, I think we should mention the following:
* ELSER V1 has worse throughput and worse retrieval performance.
* ELSER V1 embeddings are not compatible with ELSER V2 embeddings, thus migrating from ELSER V1 to ELSER V2 requires reindexing from source.
* We don't recommend ELSER V1 for new users of ELSER; ELSER V1 is only recommended for users who want to use semantic search and are unwilling or unable to upgrade to v8.11.

Reply:
As in the latest commit, all references to ELSER V1 are removed, I did not add these bits to this page. However, they are all documented in the ELSER conceptual page [here](https://www.elastic.co/guide/en/machine-learning/master/ml-nlp-elser.html#elser-v2) and [here](https://www.elastic.co/guide/en/machine-learning/master/ml-nlp-elser.html#upgrade-elser-v2).